Purchases/Purchasing/Package.swift

```diff
@@ -66,6 +69,9 @@ private extension PackageType {
     }
 }
 
+/**
+ Enumeration of all possible Package types.
```

Comment:
I think you copied this from `PackageType` :)

Reply:
I've gone through all these missing docs and fixed them, so ignore this one :)
#1231

Reply:
thanks so much for taking care of that!!